fala/settings/base.py

```diff
@@ -99,7 +109,7 @@ def root(*x):
 
 INSTALLED_APPS = ("django.contrib.sessions", "django.contrib.messages", "django.contrib.staticfiles", "requests")
 
-PROJECT_APPS = ("adviser", "laalaa")
+PROJECT_APPS = ("adviser", "fala", "laalaa")
```

Comment:
Is this adding a new fala app? I can't seem to find a fala app inside the `apps` directory?

Reply:
It's adding the main app of the project itself (ie the `fala` dir which `apps` is inside)